### ISSUE TITLE
Forward msg option request

### ISSUE THREAD

**Issue by boina** [mozillahispano/openwapp#159]
Obviously this is not a bug. I would love to be able to forward msg from one conversation to a different one.

Thanks!!

**Comment by JayVii** [comment]
indeed. the lack of copy-&-paste on older FxOS devices, makes forwarding a valuable feature
